src/compile/mark/init.ts

```diff
@@ -29,14 +31,15 @@ export function initMarkDef(mark: Mark | MarkDef, encoding: Encoding<string>, sc
 /**
  * Initialize encoding's value with some special default values
  */
-export function initEncoding(mark: Mark, encoding: Encoding<string>, stacked: StackProperties, config: Config): Encoding<string> {
+export function initEncoding(mark: Mark, encoding: Encoding<string>, stacked: StackProperties, config: Config, scales: Dict<Scale>): Encoding<string> {
```

Comment:
Seems like you no longer to modify the parameter of this method / any import in this file.